memcached-operator/deploy/olm-catalog/memcached-operator/0.11.0/memcached-operator.v0.11.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,170 @@
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "cache.example.com/v1alpha1",
+          "kind": "Memcached",
+          "metadata": {
+            "name": "example-memcached"
+          },
+          "spec": {
+            "size": 3
+          }
+        }
+      ]
+    capabilities: Basic Install
+  name: memcached-operator.v0.11.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+      - description: Represents a cluster of Memcached apps
+        displayName: Memcached App
+        kind: Memcached
+        name: memcacheds.cache.example.com
+        resources:
+          - kind: Deployment
+            name: ""
+            version: v1
+          - kind: Service
+            name: ""
+            version: v1
+          - kind: ReplicaSet
+            name: ""
+            version: v1
+          - kind: Pod
+            name: ""
+            version: v1
+        specDescriptors:
+          - description: The desired number of member Pods for the deployment.
+            displayName: Size
+            path: size
+            x-descriptors:
+              - urn:alm:descriptor:com.tectonic.ui:podCount
+        statusDescriptors:
+          - description: The current status of the application.
+            displayName: Status
+            path: phase
+            x-descriptors:
+              - urn:alm:descriptor:io.kubernetes.phase
+          - description: Explanation for the current status of the application.
+            displayName: Status Details
+            path: reason
+            x-descriptors:
+              - urn:alm:descriptor:io.kubernetes.phase:reason
+        version: v1alpha1
+  description: Main enterprise application providing business critical features with
+    high availability and no manual intervention.
+  displayName: Memcached Application
+  install:
+    spec:
+      deployments:
+        - name: memcached-operator
+          spec:
+            replicas: 1
+            selector:
+              matchLabels:
+                name: memcached-operator
+            strategy: {}
+            template:
+              metadata:
+                labels:
+                  name: memcached-operator
+              spec:
+                containers:
+                  - command:
+                      - memcached-operator
+                    env:
+                      - name: WATCH_NAMESPACE
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.annotations['olm.targetNamespaces']
+                      - name: POD_NAME
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.name
+                      - name: OPERATOR_NAME
+                        value: memcached-operator
+                    image: quay.io/example/memcached-operator:v0.0.1
+                    imagePullPolicy: Never
```

Comment:
The other CSVs in this bundle use:
```
image: REPLACE_IMAGE
imagePullPolicy: Always
```

Any reason we aren't continuing using the same convention? Thoughts on this change @estroz?

Reply:
I'd keep what the other CSV's have. We should discuss if/how we want to build and tag memcached-operator images for this repo. IMO we should get CI set up and push a new image on each release to be referenced by the CSV.

Reply:
Hi @estroz, 

It shows a great idea. But it would be out of the scope of this PR now. 
I think we need to address many improvements here.